tests/tests/cpld_2threads_p7

```diff
@@ -51,11 +51,16 @@ export LIST_FILES="sfcf024.tile1.nc \
                    RESTART/sfc_data.tile5.nc \
                    RESTART/sfc_data.tile6.nc \
                    RESTART/MOM.res.nc \
-                   RESTART/iced.2021-03-23-21600.nc \
-                   RESTART/ufs.cpld.cpl.r.2021-03-23-21600.nc"
+                   RESTART/iced.2021-03-22-64800.nc \
+                   RESTART/ufs.cpld.cpl.r.2021-03-22-64800.nc \
+                   20210322.180000.out_grd.glo_1deg \
+                   20210322.180000.out_pnt.points \
+                   20210322.180000.restart.glo_1deg"
 
 export_fv3
 export_cpl
+export FHMAX=12
```

Comment:
Is there any issue with 24 hr forecast length?

Reply:
The FHMAX=12 was in the old cpld_control_wave_p7 test and therefore carried over in the new cpld_control_p7 (with waves) and the associated tests.  This could certainly be changed if needed.

Reply:
I thought you had perhaps changed FHMAX for timing reasons. Shorter tests are generally better, but 24h is our standard length. What do you think @junwang-noaa ?

Reply:
I think it was shortened in the cpld_control_wave_p7 test  a while ago for timing reasons; however, with the lighter WW3 grid, I don't think you'll have the timing issues for the reason it was changed to 12 (assuming my memory is correct, I didn't go back and look up the commit histories to confirm).

Reply:
Let's change it back to 24hrs, the 12hr forecast with the lighter ww3 grid now takes 2mins, so I don't think it would be an issue.